packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -484,7 +488,14 @@ class SizeTransition extends AnimatedWidget {
   /// A value of 1.0 indicates the bottom or end, depending upon the [axis].
   ///
   /// A value of 0.0 (the default) indicates the center for either [axis] value.
```

Comment:
Add something like "This property has been deprecated and superseded by [alignment]. Existing usages can be migrated to [alignment] as ... ".